sandbox/main.lua

```diff
@@ -21279,6 +21272,17 @@ function createPlayerShipYorik()
 	playerYorik:addReputationPoints(50)
 	return playerYorik
 end
+function createPlayerShipSzpieg()
```

Comment:
I'm not sure that simply spawning a player ship with a template that has not been marked as a player ship will work. Have you tested this?

Reply:
This was tested during session, players piloted these ships without major issues.

Reply:
Minor issues? Well... they didn't have any rooms in engineering, so they couldn't repair stuff. And they could power on shields which did nothing (these Ktlitan vessels don't have shields. The shield powering on sound would be played but with no actual effect). 

But other than that it was fine.